rtv/submission.py

```diff
@@ -71,15 +71,14 @@ def open_link(self):
         url = self.content.get(-1)['permalink']
         open_browser(url)
 
-    @SubmissionController.register('c')
-    def add_comment(self):
+    def manipulate_comment(self, command):
```

Comment:
I don't think that this currently works well as a single function. There are too many 
```python
if command == 'c':
    ...
elif command == 'e':
   ...
else
```
style blocks. You should factor out some of the "special cases" and place them back into their parent functions. I'll try to give some examples.